src/astarte/handler.rs

```diff
@@ -153,6 +154,12 @@ pub struct Subscriber {
     sender: Sender<Result<AstarteMessage, Status>>,
 }
 
+impl Debug for Subscriber {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        write!(f, "{:?}", self.introspection)
+    }
+}
+
```

Comment:
We should derive debug and if you need to keep this, turn it into a display instead.